### ISSUE TITLE
Is it possible to send messages to a default_exchange?

### ISSUE THREAD

**Issue by DanilParshukov** [llxff/coney#5]
Hello, my problem is that I want to subscribe on a default exchange but when I put the name of the default exchange (empty string) to configs, I receive the following error
```shell
14:54:19.313 [error] GenServer #PID<0.1290.0> terminating
** (stop) 'stopping because dependent process <0.1287.0> died: {{shutdown,\n                                                      {server_initiated_close,\n                                                       403,\n                                                       <<"ACCESS_REFUSED - operation not permitted on the default exchange">>}},\n                                                     {gen_server,call,\n                                                      [<0.1299.0>,\n                                                       {call,\n                                                        {\'exchange.declare\',\n                                                         0,<<>>,<<"direct">>,\n                                                         false,false,false,\n                                                         false,false,[]},\n                                                        none,<0.1287.0>},\n                                                       60000]}}'
Last message: {:EXIT, #PID<0.1287.0>, {{:shutdown, {:server_initiated_close, 403, "ACCESS_REFUSED - operation not permitted on the default exchange"}}, {:gen_server, :call, [#PID<0.1299.0>, {:call, {:"exchange.declare", 0, "", "direct", false, false, false, false, false, []}, :none, #PID<0.1287.0>}, 60000]}}}
State: {:state, :amqp_network_connection, {:state, #Port<0.606>, "client 127.0.0.1:55513 -> 127.0.0.1:5672", 10, #PID<0.1293.0>, 131072, #PID<0.1289.0>, :undefined, false}, #PID<0.1292.0>, {:amqp_params_network, "guest", "guest", "/", 'localhost', 5672, 2047, 0, 10, 60000, :none, [#Function<12.131604370/3 in :amqp_uri.mechanisms/1>, #Function<12.131604370/3 in :amqp_uri.mechanisms/1>], [], []}, 2047, [{"capabilities", :table, [{"publisher_confirms", :bool, true}, {"exchange_exchange_bindings", :bool, true}, {"basic.nack", :bool, true}, {"consumer_cancel_notify", :bool, true}, {"connection.blocked", :bool, true}, {"consumer_priorities", :bool, true}, {"authentication_failure_close", :bool, true}, {"per_consumer_qos", :bool, true}, {"direct_reply_to", :bool, true}]}, {"cluster_name", :longstr, "rabbit@sadf"}, {"copyright", :longstr, "Copyright (C) 2007-2018 Pivotal Software, Inc."}, {"information", :longstr, "Licensed under the MPL.  See http://www.rabbitmq.com/"}, {"platform", :longstr, "Erlang/OTP 21.1.1"}, {"product", :longstr, "RabbitMQ"}, {"version", :longstr, "3.7.8"}], :none, false}
```

`ACCESS_REFUSED - operation not permitted on the default exchange`.  
I suppose that it happens because `Coney.RabbitConnection` is trying to bind queue to default exchange here: https://github.com/llxff/coney/blob/12b089dece6f28d4d9ee1181f3e52f8ec8ffbdcd/lib/coney/rabbit_connection.ex#L38 but according to this [article](https://www.rabbitmq.com/tutorials/amqp-concepts.html) we don't need to do it. 

If I'm right, and there is no way to subscribe on default exchange, can I please take this issue and fix it?
Thank you!

**Comment by llxff** [comment]
Hi, Danil! Sure, your fix is more than welcome)

**Comment by DanilParshukov** [comment]
@llxff please, check this [#pull request](https://github.com/llxff/coney/pull/6)

**Comment by llxff** [comment]
fixed in #6
